modules/nw-sriov-about-qinq.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assembly:
+//
+// * networking/hardware_networks/configuring-sriov-qinq-support.adocance/configuring-sriov-qinq-support.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-about-qinq-support_{context}"]
+= About 802.1q-in-802.1q support
```

Comment:
Per related discussion in https://github.com/openshift/openshift-docs/pull/76252#discussion_r1617520469, would you like to sync this PR up with the other's capitalization? (But not for when it's a value like `802.1q` in backticks like on [line 82](https://github.com/openshift/openshift-docs/pull/74534/files#diff-8ffa1f0cd73fa10b0cbf8fb300230d319cf306c5953c22af4e1e7a4b58741962R82))
```suggestion
= About 802.1Q-in-802.1Q support
```